macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -193,6 +193,10 @@ class TestButtonViewController: NSViewController, NSMenuDelegate {
 		let customTitle = "Atom"
 		let customImage = NSImage(named: TestButtonViewController.nonTemplateImage)!
 		customImage.isTemplate = true
+		let imageForSelectedState = NSImage(named: NSImage.actionTemplateName)!
```

Comment:
>imageForSelectedState [](start = 6, length = 21)

can we create a new section for this behavior? i am not sure if people would understand why for selected case it will show one image and disabled state another without context and reading the this test code line by line.